topgun/k8s/k8s_suite_test.go

```diff
@@ -276,3 +276,12 @@ func getRunningWorkers(workers []Worker) (running []Worker) {
 	}
 	return
 }
+
+func cleanup(releaseName, namespace string, proxySession *gexec.Session) {
```

Comment:
```suggestion
func cleanup(proxySession *gexec.Session) {
```
The releaseName and namespace are defined in the suite itself, so makes sense to refer to them.
Also, for consistency `proxySession` could also be moved to the suite itself and the suite could implement the `AfterEach`.